server/routes/deleteObject.js

```diff
@@ -7,7 +7,11 @@ import bodyParser from 'body-parser';
 app.use(cors());
 app.use(bodyParser.json());
 import { minioClient } from '../minioConfig.js';
-import { get_user_bucket, delete_file } from '../Database_queries/queries.js';
+import {
+  get_user_bucket,
+  delete_file,
+  delete_user_socket,
```

Comment:
The import statement should be updated to reflect the new function `delete_user_socket`. Ensure that the import is correctly formatted and includes all necessary functions. 
```diff
- import { get_user_bucket, delete_file } from '../Database_queries/queries.js';
+ import { get_user_bucket, delete_file, delete_user_socket } from '../Database_queries/queries.js';
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*